src/main/java/de/siegmar/logbackgelf/SimpleJsonEncoder.java

```diff
@@ -107,7 +107,8 @@ private void appendKey(final String key) throws IOException {
      */
     @SuppressWarnings("checkstyle:cyclomaticcomplexity")
     private void escapeString(final String str) throws IOException {
-        for (final char ch : str.toCharArray()) {
+        for (int i = 0; i < str.length(); i++) {
+            char ch = str.charAt(i);
```

Comment:
Please add a final in front of `char ch` and I can approve it.

Reply:
Done